images/linux/scripts/installers/mysql.sh

```diff
@@ -9,17 +9,14 @@ source $HELPER_SCRIPTS/document.sh
 
 export ACCEPT_EULA=Y
 
-# Install MySQL Client
-apt-get install mysql-client -y
-
-# InstallMySQL database development files
-apt-get install libmysqlclient-dev -y
-
 # Install MySQL Server
 MYSQL_ROOT_PASSWORD=root
 echo "mysql-server mysql-server/root_password password $MYSQL_ROOT_PASSWORD" | debconf-set-selections
 echo "mysql-server mysql-server/root_password_again password $MYSQL_ROOT_PASSWORD" | debconf-set-selections
-
+debconf-set-selections <<< 'mysql-apt-config mysql-apt-config/select-server select mysql-8.0'
+wget -c https://dev.mysql.com/get/mysql-apt-config_0.8.14-1_all.deb
```

Comment:
Why do we use hardcoded version? Please parse to the latest version- https://dev.mysql.com/downloads/repo/apt/

Reply:
Thank you, you are absolutely right, hardcoded value is not good solution here. I have applied another one.